configuration/raspi-config.md

```diff
@@ -60,7 +60,9 @@ Generally speaking, `raspi-config` aims to provide the functionality to make the
 ### Change User Password
 
 The default user on Raspberry Pi OS is ```pi``` with the password ```raspberry```. You can change that here. Read about other [users](../linux/usage/users.md).
- 
+
+From the console, this can be changed with `passwd`.
```

Comment:
Isn't that already mentioned on the "users" page that's being linked to? I think this repo generally tries to avoid having duplicate information in multiple places.